classes/class-to-specials-blocks.php

```
@@ -0,0 +1,94 @@
+
+		// BLock Helpers
+		add_filter( 'lsx_to_multi_field_wrappers', array( $this, 'register_multi_field_wrappers' ) );
+	}
+
+	/**
+	 * Register the block JSON files from build/blocks/.
+	 *
+	 * @return void
+	 */
+	public function register_block_json_files() {
+		$directory = LSX_TO_SPECIALS_PATH . 'build/blocks/';
```

PHPStan (2.2.6)

[error] Constant LSX_TO_SPECIALS_PATH not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)